managed/CounterStrikeSharp.API/Modules/Commands/Targeting/Target.cs

```diff
@@ -124,4 +111,24 @@ public TargetResult GetTarget(CCSPlayerController? caller)
         
         return new TargetResult() { Players = players };
     }
+
+    public CCSPlayerController? GetClientAimTarget(CCSPlayerController player)
+    {
+        CCSGameRules? GameRules = Utilities.FindAllEntitiesByDesignerName<CCSGameRulesProxy>("cs_gamerules").FirstOrDefault()?.GameRules;
+
+        if (GameRules == null)
+        {
+            return null;
+        }
+
+        VirtualFunctionWithReturn<IntPtr, IntPtr, IntPtr> findPickerEntity = new(GameRules.Handle, 28);
```

Comment:
I dont really like this vtable offset hardcoded, should be in the gamedata instead

Reply:
> I dont really like this vtable offset hardcoded, should be in the gamedata instead

This code is a code shared in the CSS discord. Of course, it would be better to be developed and shared.

Reply:
could you add it to the gamedata tho?
also are you sure that 28 is the offset for both platforms?